atomium/templates/atomium_overview/atomium_overview.component.inc

```diff
@@ -147,7 +153,27 @@ function atomium_preprocess_atomium_overview(array &$variables, $hook) {
         '#type' => FALSE,
       );
     }
+
+    $links[] = l(
```

Comment:
No `l()` calls, use a render array please :-)

Reply:
Why? "l" is a Drupal function which uses link theme.

Reply:
We never use it, it's against the way Atomium is working, we only do render() in templates.